keras_hub/src/models/backbone.py

```diff
@@ -277,3 +277,21 @@ def load_lora_weights(self, filepath):
             layer.lora_kernel_a.assign(lora_kernel_a)
             layer.lora_kernel_b.assign(lora_kernel_b)
         store.close()
+
+    def export_to_transformers(self, path, verbose=True):
+        """Export the backbone model to HuggingFace Transformers format.
+
+        This saves the backbone's configuration and weights in a format
+        compatible with HuggingFace Transformers. For unsupported model
+        architectures, a ValueError is raised.
+
+        Args:
+            path: str. Path to save the exported model.
+            verbose: bool. If True, print success messages (default: True).
+
+        """
+        from keras_hub.src.utils.transformers.export.hf_exporter import (
+            export_backbone,
+        )
+
+        export_backbone(self, path, verbose=verbose)
```

Comment:
let's add some high level testing of the exposed API in backbone_test.py, task_test.py, tokenizer_test.py, next to save to preset. Doesn't need to be as thorough as the gemma_test below. We just want to test the high level usage.

E.g. that you can call this function without error for a support model, and it errors for models that do not yet support this.